Content.Server/Chat/Commands/CollectiveMindCommand.cs

```diff
@@ -0,0 +1,43 @@
+using Content.Server.Chat.Systems;
+using Content.Shared.Administration;
+using Robust.Server.Player;
+using Robust.Shared.Console;
+using Robust.Shared.Enums;
+
+namespace Content.Server.Chat.Commands
+{
+    [AnyCommand]
+    internal sealed class CollectiveMindCommand : IConsoleCommand
```

Comment:
try updating this to the new `ToolshedCommand` API, its a lil recent so feel free to ask anything